src/Hintman/HLint.hs

```diff
@@ -92,28 +108,62 @@ createFileDownloadUrl PrInfo{..} (toText -> file) = T.intercalate "/"
     , file
     ]
 
-{- | Creates the comment text from the HLint 'Idea's.
+-- | Create a 'Comment' from all necessary data.
+createComment :: FileDeltas -> FilePath -> Text -> Idea -> Maybe Comment
+createComment _fd (toText -> commentPath) content idea =
+    createCommentText content idea >>= \commentBody -> Just Comment
+        { commentPosition = 0 -- TODO: get this from line number
+        , ..
+        }
+
+{- | Creates the comment text from the HLint 'Idea's and file content.
 -}
-createComment :: Idea -> Maybe Text
-createComment Idea{..} = ideaTo >>= \to ->
+createCommentText :: Text -> Idea -> Maybe Text
+createCommentText content Idea{..} = ideaTo >>= \to ->
     if ideaSeverity == Ignore
     then Nothing
-    else Just $ unlines $
+    else buildWholeLine to >>= \newLine -> Just $ unlines $
         (if ideaHint == "" then "" else show ideaSeverity <> ": " <> toText ideaHint)
```

Comment:
@chshersh maybe we should write the `ideaSeverity` anyway without this check?

Reply:
@vrom911 No, I think that the current behavior is okay. If something is ignored, people probably don't want to see it. But later we can implement `ShowIgnore` flag that will display ignores as well if somebody wants.